### DISCUSSION TITLE
How do you organize dashboards without sub folders?

### DISCUSSION THREAD

**Discussion by MA-MacDonald** [grafana/grafana#49765] [General]
Hello one of the big pain points I have with Grafana is the lack of options when it comes to organizing many dashboards.

Ideally the option for sub folders would be great but I have read the devs don't want to implement this for some reason.
I work in a large org with 60,000+ employees and we have many dashboards for various teams and applications.

As the Grafana admin for my org I am struggling to find a good way to organize and maintain a large collection of dashboards.

I would like to know how some of the other large orgs handle this?

For example do you create folders based on teams or based on the application or both?

Ideally I would love to be able to create a folder for each team and within the team folder have application level folders but this is not possible.

--Team 1
------App1
---------Dashboard 1
---------Dashboard 2
------App2
---------Dashboard 1
---------Dashboard 2
--Team 2
------App1
---------Dashboard 1
---------Dashboard 2

**Comment by torkelo** [comment]
You also have tags that you can use.

We are working on ways to add nested folders

**Reply by MA-MacDonald** [reply]
Thanks for the update, I'm glad to hear nested folders are being worked on!

Also we do use tags they just aren't my favorite way to organize things ;P

Thanks for all your hard work, you guys rock!

**Comment by jdmaloney** [comment]
I just wanted to drop a note in this discussion to say we'd greatly enjoy nested folders as well.  We use folders (the one level we get) and tags but having the ability to nest folders would be great.  We have some folders that have 50+ dashboards in them and being better able to further break those down would be fantastic.  

Edit:
@MA-MacDonald One thing we also leverage is the dashboard list panel; we actually have a dashboard that has a bunch of those panels that gives us a place for folks to more easily (somewhat) find dashboards they're looking for.

**Reply by MA-MacDonald** [reply]
Ah so you basically make a dashboard as a dedicated index. That could be useful thanks for the tip!

**Comment by pmohan6** [comment]
Is there any update on this story? We'd love to see nested folders as well!

**Comment by Aracki** [comment]
In Grafana 10, one can just turn on `nestedFolders` feature.

**Reply by cicciopizza** [reply]
Where ?

**Reply by cicciopizza** [reply]
oopss...

[feature_toggles]
**nestedFolders = true**
;enable = feature1,feature2

;feature1 = true
;feature2 = false

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
